messages/definitions.json

```diff
@@ -478,30 +507,78 @@
         "unit": {
           "type": "string",
           "title": "Event unit",
-          "description": "The unit of the event"
+          "description": "The unit of the event (deprecated in favour of data.unit)"
```

Comment:
As a pointer to the future, the latest JSON Schema spec supports deprecated:
https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.9.3

Reply:
Added.